url.go

```diff
@@ -89,6 +89,17 @@ func ParseURL(str string) (opt ClientOption, err error) {
 		_, addr = parseAddr(addr)
 		opt.InitAddress = append(opt.InitAddress, addr)
 	}
+	if opt.TLSConfig != nil && q.Has("skip_verify") {
+		if q.Get("skip_verify") == "" {
+			opt.TLSConfig.InsecureSkipVerify = true
+		} else {
+			skipVerify, err := strconv.ParseBool(q.Get("skip_verify"))
+            if err != nil {
+				return opt, fmt.Errorf("valkey: invalid skip verify: %q", q.Get("skip_verify"))
+			}
+			opt.TLSConfig.InsecureSkipVerify = skipVerify
+		}
```

Comment:
```suggestion
		if v := q.Get("skip_verify"); v == "" {
			opt.TLSConfig.InsecureSkipVerify = true
		} else {
			skipVerify, err := strconv.ParseBool(v)
			if err != nil {
				return opt, fmt.Errorf("valkey: invalid skip verify: %q", v)
			}
			opt.TLSConfig.InsecureSkipVerify = skipVerify
		}
```

Reply:
LGTM mostly, but the indention on line 97 is wrong, and one minor improvement is you can store the result of q.Get() into a variable.

Reply:
Suggestions applied